test/FigletTest.php

```diff
@@ -118,9 +119,9 @@ public function testGzippedFont()
         $this->_equalAgainstFile($figlet->render('Dummy'), 'StandardAlignLeft.figlet');
     }
 
-    public function testConfig()
+    public function testArrayAccessConfig()
     {
-        $config = new \Laminas\Config\Config(['justification' => Figlet\Figlet::JUSTIFICATION_RIGHT]);
+        $config = new ArrayObject(['justification' => Figlet\Figlet::JUSTIFICATION_RIGHT]);
```

Comment:
I propose this change because the `Figlet` constructor is only looking for `Traversable`; it then uses the laminas-stdlib `ArrayUtils::iteratorToArray()` functionality to cast it to an array before using the value to set options. As such, usage of laminas-config for the test is unnecessary.